### PR TITLE
Updated Ranks and Authorized Assets

### PR DESCRIPTION
**Pull request by Xrez31**

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -15,15 +15,15 @@ Config.AuthorizedVehicles = {
 	car = {
 		crew = {},
 
-		cameraman = {
+		cam_man = {
 			{model = 'rumpo', price = 20000}
 		},
 
-		reporter = {
+		report = {
```
Comment (alvinveroy) [inline config.lua]: See
https://github.com/lockdownstudio/esx_journalist/blob/17947aad0717575854d6250436465a117d64f476/client/main.lua#L140

This changes will throw error on the client script.

**Review by alvinveroy — CHANGES_REQUESTED** [review]
Changes need to be done in the client.lua file or retain the original job name.

**Comment by alvinveroy** [comment]
Put into consideration this script needed as props for Weazel News
https://github.com/lockdownstudio/WeazelNewsCam

/bmic for job grade 0
/cam for job grade 1
/mic for job grade 2

**Comment by Xrez31** [comment]
Fixed rank LABELS*** and SALARY only, added option for cloakroom, repositioned coords and new uniforms.

**Pull request merged**